src/Core/Migration/V6_6/Migration1747746986OrderTaxCalculationType.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_6;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+class Migration1747746986OrderTaxCalculationType extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1747746986;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $this->addColumn($connection, 'order', 'tax_calculation_type', 'varchar(255)');
```

Comment:
Length can be reduced to be the same as in [`sales_channel`](https://github.com/shopware/shopware/blob/636d59ea457b5cdf77388760830cb3638c79001c/src/Core/Migration/V6_3/Migration1589178550AddTaxCalculationType.php#L24):
```suggestion
        $this->addColumn($connection, 'order', 'tax_calculation_type', 'varchar(50)');
```